### PR TITLE
Refs #7806 - document roles organizations and locations

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: _includes/manuals/1.13/4.1.2_roles_and_permissions.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ These may be created, deleted and edited on the **Roles** page. Each role will c
 
 There is one built-in system role, 'Default role'. This is a set of permissions that every user will be granted, in addition to any other roles that they have.
 
+Roles can be also associated to Locations or Organizations if these are allowed. Unlike other objects this does not mean that Roles would be only available in a particular scope. Roles are always global for the whole Foreman. The association means that filters of such role are scoped to a particular Organization or Location. Imagine you want to create a role representing Manager of Organization A. You can clone existing Manager role and associate it with Organization A. If you later assign this role to some users, they will be granted manager permissions but only on resources of Organization A. Note that some resource are not scopeable by Organization and Locations. Filters for such resources grants permissions globally.
```
Comment (dLobatog) [inline _includes/manuals/1.13/4.1.2_roles_and_permissions.md]: typos:

'You can clone _an_ existing Manager role'
 'filters for such resources _grant_ permissions globally'  
"note that some _resourceS_ are not scopeable".

File: _includes/manuals/1.13/4.1.2_roles_and_permissions.md [hunk 2]
```diff
@@ -33,6 +35,8 @@ Some example queries for the resource Host:
 
 These pools of queries can be combined by adding them together or the filters can be used to restrict the selected resource to a smaller and smaller subset of the total. Think of them as set operations.
 
+As already mentioned, Role can be assigned to Organizations and Locations. In such case, all filters for resources that support such scoping automatically apply the same Organizations and Locations. If you want to combine filters with different Organizations or Locations assignments, you can use 'Override' check box. When checked you can override Organizations and Location for a filter. If you uncheck this field, the filter starts inheriting its role Organizations and Locations after submitting again. If you want to reset all role's filter to start inheriting, you can use 'Disable all filters overriding' button on role's 'Filters' tab. We recommend managing Organizations and Locations association on Role level to keep the setup simple and clear.
```
Comment (dLobatog) [inline _includes/manuals/1.13/4.1.2_roles_and_permissions.md]: "_a_ Role can be assigned to"
"if you want to reset all _role filters_"

A screenshot to show where 'disable all filters overriding' is located would help IMO, it's hard to tell from just text.

**Comment by dLobatog** [comment]
Thanks @ares , two comments inline then we can merge it :+1:

**Comment by ares** [comment]
Thanks @dLobatog, updated.

**Comment by dLobatog** [comment]
Merged!

**Pull request merged**